relayer/pkg/starknet/chain_client.go

```diff
@@ -13,30 +13,30 @@ import (
 type FinalizedBlock = starknetrpc.Block
 
 // used to create batch requests
-type StarknetBatchBuilder interface {
-	RequestBlockByHash(h *felt.Felt) StarknetBatchBuilder
-	RequestBlockByNumber(id uint64) StarknetBatchBuilder
-	RequestChainId() StarknetBatchBuilder
-	// RequestLatestPendingBlock() (StarknetBatchBuilder)
-	RequestLatestBlockHashAndNumber() StarknetBatchBuilder
-	// RequestEventsByFilter(f starknetrpc.EventFilter) (StarknetBatchBuilder)
-	// RequestTxReceiptByHash(h *felt.Felt) (StarknetBatchBuilder)
+type BatchBuilder interface {
```

Comment:
yeah I guess this makes sense. it's already in starknet repo so no need to add the starknet-prefix. 

the linter didn't pick that up though right?! that's just your eye for detail i presume?